build.gradle

```diff
@@ -431,7 +449,7 @@ project(':iceberg-spark-runtime') {
     relocate 'org.apache.orc', 'org.apache.iceberg.shaded.org.apache.orc'
     relocate 'io.airlift', 'org.apache.iceberg.shaded.io.airlift'
 
-    archiveBaseName = "iceberg-spark-runtime"
+    archiveName = "iceberg-spark-runtime-${version}.${extension}"
```

Comment:
This rolls back a change from #236 that added an extra publication Jar because the shaded Jar used a classifier and no longer replaced the default.